man/resolved.conf.xml

```diff
@@ -269,6 +269,22 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes an IPv4 or IPv6 address to listen on. The address may optionally be prefixed by <literal>:</literal> and
+        a protocol name (<literal>udp</literal> or <literal>tcp</literal>). It may also be optionally suffixed by <literal>:</literal>
+        and a numeric port number. If the protocol is not specified, the service will listen on both IPv4 and IPv6. If the port is not
```

Comment:
`optionally suffixed by a numeric port number with separator <literal>:</literal>.`

`both IPv4 and IPv6` -> `both UDP and TCP`